PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -429,14 +467,21 @@
         }
 
         private func isImageViewSensitive(_ view: UIImageView) -> Bool {
-            var isAsset = false
-            if let image = view.image {
-                isAsset = isAssetsImage(image)
-            } else {
-                // if there's no image, there's nothing to mask
-                return false
+            // if there's no image, there's nothing to mask
+            guard let image = view.image else { return false }
+
+            // sensitive, regardless
+            if view.isNoCapture() {
```

Comment:
this should be the case already in L353 right? we are doing 2 checks for the very same view.

Reply:
true that, maybe `view.isNoCapture()` should be the very first thing to be done.
they are on android https://github.com/PostHog/posthog-android/blob/00df23213f574344871c2e16cca3dc16e3aa2a41/posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt#L542-L549
so we don't need to repeat them.